audit/projects/k8s-staging-ci-images/services/enabled.txt

```diff
@@ -3,7 +3,6 @@ cloudbuild.googleapis.com         Cloud Build API
 cloudkms.googleapis.com           Cloud Key Management Service (KMS) API
 containeranalysis.googleapis.com  Container Analysis API
 containerregistry.googleapis.com  Container Registry API
-containerscanning.googleapis.com  Container Scanning API
```

Comment:
Expected, this is https://github.com/kubernetes/k8s.io/issues/1963#issuecomment-844588412